frontend/src/scenes/notebooks/Notebook/SlashCommands.tsx

```diff
@@ -449,7 +457,7 @@ export const SlashCommands = forwardRef<SlashCommandsRef, SlashCommandsProps>(fu
     }
 
     return (
-        <div className="SlashCommands space-y-px">
+        <div className="space-y-1">
```

Comment:
Why this change? With Menus, we typically only have a px space, not a 1 (0.25rem)
```suggestion
        <div className="space-y-px">
```

Reply:
The vertical margin looked incredibly tight to me at 1px because of the hover state on the buttons. The horizontal padding was also different
|Before|After|
|----|----|
| <img width="262" alt="Screenshot 2023-10-13 at 10 39 12" src="https://github.com/PostHog/posthog/assets/6685876/1f6fafa4-8478-4c9a-ab8d-eb44147ede55"> | <img width="270" alt="Screenshot 2023-10-13 at 10 37 27" src="https://github.com/PostHog/posthog/assets/6685876/07e3c8c2-c1c8-4758-bd62-418e43af0928"> |

Reply:
I'm still tempted to push back, mainly because we are doing something here different to everywhere else. 

If we agree it's too tight then fine - change it everywhere, but weird we would have more padding here than any other similar UI

Reply:
Fair point. The thing for me here is that the horizontal divider isn't something we use in other similar UI (e.g. LemonMenu). Where we do use one (e.g. in the `NotebookSelectButton`) the spacing of everything is on a 0.25rem basis. Will fix forwards in that case rather than changing this.

Ultimately we should probably decide at a design system level what impact dividers have on a menu but that's for another day